test/integration/targets/get_url/tasks/main.yml

```diff
@@ -761,3 +761,16 @@
 - assert:
     that:
       - get_dir_filename.dest == remote_tmp_dir ~ "/filename.json"
+
+- name: Test for incomplete data read (issue 85164)
```

Comment:
@Shrews this might need some preparation step waiting for the server to be up. Or just retries. It sometimes gets a "Connection refused" instead and I think it's because of a race condition of the server not starting fast enough on slow systems like macos..
https://dev.azure.com/ansible/ansible/_build/results?buildId=159060&view=logs&j=f59ce2ad-d869-5f6f-71a8-430923834cee&t=40484b38-0454-5ba9-2f2a-9f86c8eb94ee&l=7182